lib/v8.js

```diff
@@ -478,4 +502,6 @@ module.exports = {
   setHeapSnapshotNearHeapLimit,
   GCProfiler,
   isStringOneByteRepresentation,
+  startCpuProfile,
+  stopCpuProfile,
```

Comment:
I'm not a huge fan of the API here but won't block on this point. I'd much rather see a pattern like:

```js
const profile = v8.startCpuProfile('foo');
// .. do stuff
profile.stop();
```

or, with explicit resource management

```js
{
  using profile = v8.startCpuProfile('foo');
  // .. do stuff
}
// profile is stopped when the scope exits
```